herd/AArch64ASLSem.ml

```diff
@@ -445,62 +473,30 @@ module Make (TopConf : AArch64Sig.Config) (V : Value.AArch64ASL) :
           in
           None
 
-    let symb_values_tbl : (string, V.v) Hashtbl.t = Hashtbl.create 17
-    let symb_values_tag = "AArch64-HashedValue"
+    let tr_cst tr =
+      Constant.map tr
+        (fun _ -> Warn.fatal "Cannot translate PTE")
+        (fun _ -> Warn.fatal "Cannot translate instruction")
+
+    let aarch64_to_asl_bv = function
+      | V.Var v -> ASLS.A.V.Var v
```

Comment:
Will there be a problem if the symb counter on ASL values is not increased and we have symbolic values with the same number?
I think we should use fresh_var and keep track of why ASL value equals which AArch64 value, or make the counter of symbolic values common for all value types.

Reply:
Yes it can be a problem if two different variables are the same.  This can be the case, I am afraid.

Reply:
A new commit 5d08b1cc82e379a11a717cacc7f0bb131f2d95db intend to fix this potential problem.